apps/rich-text-app/src/index.tsx

```diff
@@ -1,24 +1,24 @@
 import * as React from 'react';
 import { render } from 'react-dom';
-import { init, FieldExtensionSDK, locations } from '@contentful/app-sdk';
+import { FieldExtensionSDK, DialogExtensionSDK, init, locations } from '@contentful/app-sdk';
```

Comment:
As we are already changing things here: `FieldAppSDK` and `DialogAppSDK` are the new names for these types. The old types will be removed in App SDK v5

Reply:
We don't have this types here yet ("@contentful/app-sdk": "^4.6.0") and I assume we have to do this upgrade then also for all the Editors?
So would vote to do this in a separate ticket

Reply:
True, I assumed you upgraded to latest

-> another PR